### PR TITLE
Add SGraph.show with vertex coloring

### PR DESCRIPTION
**Pull request by znation**

### CODE REVIEW

File: sgraph_show_with_vertex_coloring.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+             [0.7803921568627451, 0.7803921568627451, 0.7803921568627451],
+             [0.7372549019607844, 0.7411764705882353, 0.13333333333333333],
+             [0.8588235294117647, 0.8588235294117647, 0.5529411764705883],
+             [0.09019607843137255, 0.7450980392156863, 0.8117647058823529],
+             [0.6196078431372549, 0.8549019607843137, 0.8980392156862745]]
+
+# Add some example edges -- replace with your own graph
+g = nx.dodecahedral_graph()
+sg = gl.SGraph()
+sg = sg.add_vertices([gl.Vertex(i) for i in g.nodes()])
```
Comment (papayawarrior) [inline sgraph_show_with_vertex_coloring.py]: This is a great example of how well we work with other tools - I dig!

File: sgraph_show_with_vertex_coloring.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Show an SGraph with custom vertex coloring
+import graphlab as gl
+import networkx as nx # Requires 'pip install networkx==1.9.1'
+
+# We will use the "Tableau 20" color scheme as RGB triples scaled to 0-1
+# See http://public.tableausoftware.com/profile/chris.gerrard#!/vizhome/TableauColors/ColorPaletteswithRGBValues
+tableau20 = [[0.12156862745098039, 0.4666666666666667, 0.7058823529411765],
```
Comment (papayawarrior) [inline sgraph_show_with_vertex_coloring.py]: Might it be possible to pull this in automatically rather than enumerating it in the how-to?

**Comment by srikris** [comment]
@znation Good to go. Rebase and merge.

File: sgraph_show_with_vertex_coloring.py (hunk 2 above)
Comment (znation) [inline sgraph_show_with_vertex_coloring.py]: I don't think there is an easy way to get these values programatically. They are mostly just described on the web (in HTML) which would take more code to parse (unreliably) than to just embed the values here.

**Pull request merged**